MaterialBudget.cpp

```diff
@@ -2,20 +2,20 @@
 
 ClassImp(MaterialBudget)
 
-MaterialBudget::MaterialBudget(): fHeight(0.), fRadius(0.), fLength(0.), fDensity(0.), fZ(0), fA(0){}
+MaterialBudget::MaterialBudget(): fHeight(0.), fRadius(0.), fLength(0.), fDensity(0.), fZ(0), fA(0), fMultScat(0){}
 
-MaterialBudget::MaterialBudget(double height, double radius, double length, double density, int z, int a)
+MaterialBudget::MaterialBudget(double height, double radius, double length, double density, int z, int a, bool multscat)
 {
     if(height<=0) cout << "Material Budget height null or negative. Setting value equal to 1. \n";
     height = 1.;
 
     if(radius<=0) cout << "Material Budget radius null or negative. Setting value equal to 1. \n";
     radius = 1.;
 
-    SetGeometry(height,radius, 0.27).SetMaterial(density, z, a);
+    SetGeometry(height,radius, 0.27).SetMaterial(density, z, a).SetStatus(multscat);
 }
 
-MaterialBudget::MaterialBudget(double height, double radius, double length, string material)
+MaterialBudget::MaterialBudget(double height, double radius, double length, bool multscat, string material)
```

Comment:
I think multscat should go after material (as in the previous funcion definitio)